util/http.go

```diff
@@ -8,3 +11,14 @@ func SetCookie(w http.ResponseWriter, cookie *http.Cookie) {
 	cookie.Secure, cookie.HttpOnly = true, true
 	http.SetCookie(w, cookie)
 }
+
+func RequestMethodCheck(w http.ResponseWriter, r *http.Request, methods ...string) bool {
```

Comment:
Referring to my last comment, please call it `checkMethod`. Verb first, because it is a `func`, and omit "request" in the name because this is obvious (a response has no method).

Also add documentation.